dev.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:1.81.0-alpine3.20
+
+RUN apk add --no-cache --update \
+    docker=26.1.5-r0 nodejs-current=21.7.3-r0 \
+    musl-dev=1.2.5-r0 build-base=0.5-r3 \
+    npm=10.8.0-r0 libssl3=3.3.2-r0 openssl-libs-static=3.3.2-r0 \
+    && rm -rf /var/cache/apk/* && cargo install cargo-watch
```

Comment:
Por tema de legibilidad y cache de `steps` sugiero evitar tener los comandos en un solo, yo lo dejaria asi

```dockerfile
RUN apk add --no-cache --update \
    docker=26.1.5-r0 nodejs-current=21.7.3-r0 \
    musl-dev=1.2.5-r0 build-base=0.5-r3 \
    npm=10.8.0-r0 libssl3=3.3.2-r0 openssl-libs-static=3.3.2-r0
RUN rm -rf /var/cache/apk/*
RUN cargo install cargo-watch
```

Reply:
Hacer eso no es bueno, habia una warning al respecto, pero ahora encontre eso

https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-which-is-better

Reply:
@stifskere Pasa que en este caso estas metiendo toda la carga de instalacion a un solo paso, esto lo hace lento en un segundo rebuild ademas de que no es muy legible, por lo menos el `cargo install cargo-watch` deberia ser otro `RUN`

Reply:
Ese `cargo install cargo-watch` ya no existe, entonces eso lo daria por solucionado